src/opl.c

```diff
@@ -1766,14 +1774,13 @@ static void init(void)
     while (!padStatus)
         padStatus = startPads();
     readPads();
-    if (!getKeyPressed(KEY_START)) {
+    if (!(getKeyPressed(KEY_TRIANGLE) && getKeyPressed(KEY_CROSS))) {
```

Comment:
I am not sure that it will work as expected. Did you check this on real hardware?

Reply:
Hm, yeah, I have tested this on my Fat SCPH-39003, and this seemed to work as expected - holding Triangle and Cross on boot resets the Video Mode to auto. Is there something you're concerned about in particular?

Reply:
by the looks of it that would be because you've changed it from start to triangle and cross.. so if those buttons are held it doesn't load user config and instead uses defaults, default vmode is auto.. that's all this section of code is doing

Reply:
Well, that's rather embarrassing - mea culpa, a silly mistake. I've resolved this issue.

Reply:
> huh, I just were not sure that it will recognise that 2 buttons are pressed simultaneously by calling getKeyPressed, but if it can recognize 2 buttons than it is ok to me now.

yea I'm not sure either.. I know what I said above is correct but idk about multiple buttons are pressed simultaneously.. easy way to test is try it just holding triangle and see if it defaults vmode

Reply:
I've just checked this by holding just Triangle or Cross and neither triggers the reset - I wouldn't really have expected this to, if getKeyPressed could only recognise a single keypress at once then surely we'd end up in the situation where we never trigger the mode reset. Either way, it seems to me like this does indeed work. 

I had thought that the enums supplied to getKeyPressed could be &'d and it'd be checking a bitfield to see if it was pressed, but from my initial observation of the function it seemed like that wasn't the case. If there's a better solution you'd like me to use I'd be happy to explore it :)

Reply:
If you tested it then it seems fine to me

Reply:
Ok, Ill merge it.